docs/custom_domains.rst

```diff
@@ -63,22 +63,26 @@ You can also host your documentation from your own domain.
          they do not yet allow us to acquire SSL certificates for those domains.
          Follow the new setup to have a SSL certificate.
 
-      .. warning:: Notes for Cloudflare users
+      .. admonition:: Certificate Authority Authorization (CAA)
```

Comment:
We should explicitly mention what record users need to add. The Cloudflare docs on this aren't super useful, having just walked a user through it. I believe we need the digicert records.

Should we consider migrating to Lets Encrypt? Everyone I've seen that has misconfigured this, has set it to LE, so it might solve a lot of the issues.

Reply:
I pushed a small patch to add this.